### ISSUE TITLE
Error while searching for marker genes

### ISSUE THREAD

**Issue by krithikaarumugam** [anuradhawick/LRBinner#10]
hi @anuradhawick,

I've been trying to bin contigs that has been assembled from ONT data using metaFlye but I get the following error.
Any help would be great! Thanks!

```
$ LRBinner_v1.0/LRBinner contigs -k 4 --reads-path ../basecalled_reads/new/all.fastq -t 44 -sep -o flye_contigs_bin_4mer --contigs ../assembly/flye/flye_CPXS12_34_56_78_910_meta/assembly_new_500.fasta

2022-07-19 16:08:26,113 - INFO - Computing contig lengths
2022-07-19 16:08:27,421 - INFO - Searching for marker genes
Traceback (most recent call last):
  File "/export2/home/krithika/tools/LRBinner/LRBinner_v1.0/LRBinner", line 197, in <module>
    main()
  File "/export2/home/krithika/tools/LRBinner/LRBinner_v1.0/LRBinner", line 179, in main
    pipelines.run_contig_binning(args)
  File "/export2/home/krithika/tools/LRBinner/LRBinner_v1.0/mbcclr_utils/pipelines.py", line 82, in run_contig_binning
    marker_gene_utils.get_contigs_with_marker_genes(output, 0.5, contig_length, 1000)
  File "/export2/home/krithika/tools/LRBinner/LRBinner_v1.0/metacoag_utils/marker_gene_utils.py", line 57, in get_contigs_with_marker_genes
    with open(f"{output}/marker_genes/contigs.hmmout", "r") as myfile:
FileNotFoundError: [Errno 2] No such file or directory: 'flye_contigs_bin_4mer/marker_genes/contigs.hmmout'
```

**Comment by Vini2** [comment]
Hello @krithikaarumugam! Thanks for raising this issue. We will fix it and update you ASAP.

**Comment by Vini2** [comment]
Hello @krithikaarumugam,

Can you check if HMMER and FragGeneScan are installed and available in your system path? 

If you are using conda, please ensure that HMMER and FragGeneScan are available in your conda environment.

Thank you!

**Comment by krithikaarumugam** [comment]
Thanks @Vini2.  Sorry about the delayed reply. I realised when running,
`python setup.py build`

there was an error when downloading FragGeneScan1.31.tar.gz from sourceforge.net.

`ERROR: cannot verify downloads.sourceforge.net's certificate, issued by ‘/C=US/O=Let's Encrypt/CN=R3’:
  Issued certificate has expired.
To connect to downloads.sourceforge.net insecurely, use `--no-check-certificate'.`

LRbinner ran without an error after I downloaded FragGeneScan1.31.tar.gz separately and moved and uncompressed it in the auxiliary/ folder.
